.github/workflows/step_test.yaml

```diff
@@ -84,6 +84,52 @@ jobs:
         with:
           workflowPreset: "${{ matrix.toolchain }}-ci"
         continue-on-error: ${{ matrix.experimental && inputs.mask-experimental}}
+  cmake-version:
+    name: >
+      🔨 CMake ${{ matrix.cmake }}
+      ${{ matrix.experimental && '[🧪 Experimental]' || '' }}
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental || false }}
+    strategy:
+      fail-fast: false
+      matrix:
+        cmake: [ "3.20", "latest", "latestrc" ]
+        include:
+          - cmake: latestrc
+            experimental: true
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install dependencies
+        run: |
+          python3 -m pip install pip-tools
+          pip-compile --extra=test --strip-extras pyproject.toml ${{ matrix.pre && '--pre' }}
+          python3 -m pip install -r requirements.txt
+      - uses: lukka/get-cmake@latest
+        with:
+          cmakeVersion: ${{ matrix.cmake }}
+      - name: Run CMake workflow
+        uses: lukka/run-cmake@v10.3
+        with:
+          workflowPreset: gcc-ci
+        continue-on-error: ${{ matrix.experimental && inputs.mask-experimental}}
+        if: matrix.cmake != '3.20'
```

Comment:
It would be nice to store "3.20" as variable.

Reply:
Github worklows does not have variables except for the inputs, so at most I can move it to `ci.yaml`. But I can add a step to extract the minimum cmake version. A simple `grep | sed`

Reply:
I see. I didn't know that 😮